### ISSUE TITLE
Add support for Web Inspector with iOS 16.4

### ISSUE THREAD

**Issue by JBYoshi** [mozilla-mobile/firefox-ios#13726]
In iOS 16.4 (currently in beta but due to be released soon), `WKWebView` has a [new property `inspectable`](https://webkit.org/blog/13936/enabling-the-inspection-of-web-content-in-apps/) that enables Safari Web Inspector support for the web view. It would be nice to have support for this so website developers can debug their sites on Firefox for iOS.

The WebKit article mentions that web inspection will also need to be turned on in the device's Safari settings for it to work on other web views, so I think it would be sufficient to simply set it to `YES` for all tabs without adding a Firefox-specific toggle.

**Comment by lmarceau** [comment]
We can add this as a debug menu option (our debug options are hidden behind 5 clicks on the version in the settings). So if the debug menu option is clicked, then the webview's would become inspectable. We create different webviews in our app, but the one we should set as inspectable is the one in Tab.swift

**Comment by gutley** [comment]
If this is implemented, please consider those of us who fork Firefox and don't want this available. E.g maybe make it contingent on the app being one of the known Mozilla bundle IDs, or perhaps a flag in the info.plist.

**Comment by OrlaM** [comment]
Sorry @gutley but we don't take forks of the repo into consideration when deciding what features to add to the project and we really don't want to go down the road of adding configuration settings for everything a fork may or may not want to adopt as the project config is already complicated enough. 
If this feature is added then the PR will be referenced in this ticket so you can always revert it on your fork if you wish to. We squash our commits so that should make it a straightforward task.

**Comment by JBYoshi** [comment]
I'm not sure if this necessarily fits with the rest of the settings in the Firefox debug menu. Those all look like they're intended for developers of the Firefox app itself, while this would also be applicable to developers not primarily focused on Firefox. I'd personally lean towards only requiring the shared iOS "Web Inspector" toggle.

**Comment by gutley** [comment]
> I'd personally lean towards only requiring the shared iOS "Web Inspector" toggle.

Agreed. Any other toggle would still require that one to be set anyway so could be confusing.
